oauthlib/oauth2/rfc6749/grant_types/base.py

```diff
@@ -116,14 +116,29 @@ def register_code_modifier(self, modifier):
     def register_token_modifier(self, modifier):
         self._token_modifiers.append(modifier)
 
-
     def create_authorization_response(self, request, token_handler):
+        """
+        :param request: oauthlib.common.Request
+        :param token_handler: A token handler instace, for example of type
+                              oauthlib.oauth2.BearerToken.
+        """
         raise NotImplementedError('Subclasses must implement this method.')
 
     def create_token_response(self, request, token_handler):
+        """
+        :param request: oauthlib.common.Request
+        :param token_handler: A token handler instace, for example of type
+                              oauthlib.oauth2.BearerToken.
+        """
         raise NotImplementedError('Subclasses must implement this method.')
 
     def add_token(self, token, token_handler, request):
+        """
+        :param token: 
+        :param request: oauthlib.common.Request
```

Comment:
Order of params should be, token, token_handler, then request :-)

Reply:
nice catch! i fixed the order in a handful of existing ones, yet somehow managed to make my own mistake too!